scripts/skimNtuple.py

```diff
@@ -202,6 +201,10 @@ def parseInputFileList (fileName) :
     tagname = "/" + opt.tag if opt.tag else ''
     jobsDir = currFolder + tagname + '/SKIM_' + basename (opt.input)
     jobsDir = jobsDir.rstrip (".txt")
+
+    if opt.EFTrew != 'none':
+        jobsDir = currFolder + tagname + '/SKIM_' + basename (opt.input) + '_EFT_'+ opt.EFTrew
+
```

Comment:
in principle, yes. but maybe it's better to have it custom instead of relying on `opt.EFTrew`. If you launch skims using the `--kl, --kt, --c2v, ...` options you end up with the same problem, no?

Reply:
good point, I'll add a check to ensure this is taken into account

Reply:
@camendola I fixed this in 25e0fa0 
please have a look

Reply:
After few tests I changed this in 76ffa31 now it should be fine.